src/wasm-common.h

```diff
@@ -298,32 +298,32 @@ enum { WASM_USE_NATURAL_ALIGNMENT = 0xFFFFFFFF };
   V(I64, I64, I64, 0, 0x8a, I64_ROTR, "i64.rotr")                       \
   V(F32, F32, F32, 0, 0x8b, F32_ABS, "f32.abs")                         \
   V(F32, F32, F32, 0, 0x8c, F32_NEG, "f32.neg")                         \
-  V(F32, F32, F32, 0, 0x8d, F32_COPYSIGN, "f32.copysign")               \
-  V(F32, F32, F32, 0, 0x8e, F32_CEIL, "f32.ceil")                       \
-  V(F32, F32, F32, 0, 0x8f, F32_FLOOR, "f32.floor")                     \
-  V(F32, F32, F32, 0, 0x90, F32_TRUNC, "f32.trunc")                     \
-  V(F32, F32, F32, 0, 0x91, F32_NEAREST, "f32.nearest")                 \
-  V(F32, F32, F32, 0, 0x92, F32_SQRT, "f32.sqrt")                       \
-  V(F32, F32, F32, 0, 0x93, F32_ADD, "f32.add")                         \
-  V(F32, F32, F32, 0, 0x94, F32_SUB, "f32.sub")                         \
-  V(F32, F32, F32, 0, 0x95, F32_MUL, "f32.mul")                         \
-  V(F32, F32, F32, 0, 0x96, F32_DIV, "f32.div")                         \
-  V(F32, F32, F32, 0, 0x97, F32_MIN, "f32.min")                         \
-  V(F32, F32, F32, 0, 0x98, F32_MAX, "f32.max")                         \
+  V(F32, F32, F32, 0, 0x8d, F32_CEIL, "f32.ceil")                       \
```

Comment:
These numbers are not matching what is in
[BinaryEncoding.md](https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md).

According to that:
f32.ceil  is 0x7e
f32.floor is 0x7f
f32.trunc is 0x80
etc.